Automattic/vip-go-mu-plugins
# Update Akismet to 5.3.1

## Description
Bumps Akismet from 5.3 to 5.3.1 released back in January.

## Changelog Description

### Plugin Updated: Akismet to 5.3.1

We upgraded Akismet 5.3 to Akismet 5.3.1. 

This version contains the following changes: 

* Make the plugin more resilient when asset files are missing (as seen in WordPress Playground).
* Add a link to the 'Account overview' page on akismet.com.
* Fix a minor error that occurs when another plugin removes all comment actions from the dashboard.
* Add the akismet_request_args filter to allow request args in Akismet API requests to be filtered.
* Fix a bug that causes some contact forms to include unnecessary data in the comment_content parameter.
*  
## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

1. Check out PR.
1. Ensure that Akismet version in WP-Admin (and `wp plugin list`) is showing 5.3.1.
1. Click around and ensure no unexpected behavior occurs and scripts are loaded on the front end.
